### DISCUSSION TITLE
Tooltip doesn't show on hover

### DISCUSSION THREAD

**Discussion by geoidesic** [carbon-design-system/carbon-components-svelte#1941] [Q&A] [answered]
### Discussed in https://github.com/carbon-design-system/carbon-components-svelte/discussions/1802

<div type='discussions-op-text'>

<sup>Originally posted by **geoidesic** September  1, 2023</sup>
https://carbon-components-svelte.onrender.com/components/Tooltip

I can appreciate that on click is useful and necessary for mobile devices but it's not useful for desktop, where the idea is that a tooltip provides information about the element you are hovering over.

The one on the IBM live demo works as expected: https://carbondesignsystem.com/components/tooltip/usage/#live-demo</div>

**Comment by brunnerh** [comment]
`carbon-components-svelte` is still on v10 of the design system while the current version is v11.
Would have to dig through the history but I think this is according to specification.

In v11 Tooltip was sort of split up into two components: `Tooltip`, which should not contain interactive content and shows on hover and [`Toggletip`](https://carbondesignsystem.com/components/toggletip/usage/), which can have interactive content and requires a click.

You can see a reference to this in the [migration guide](https://carbondesignsystem.com/migrating/guide/overview/):
> Tooltip, TooltipIcon, and TooltipDefinition are also receiving updates in v11 and mirror some of the changes seen in Notification. For example, Tooltips can now only contain interactive content. For situations that require interactive content, we are shipping a new component: Toggletip.

And the Toggletip docs:
> New in v11: Toggletip should be used in place of tooltip if your content will contain interactive elements. If it does not have any interactive content, consider using [Tooltip](https://carbondesignsystem.com/components/tooltip/usage/) instead.

---

Integration of v11 is tracked here:
- #1629

**Comment by brunnerh** [comment, marked as answer]
If you want a simple tooltip that appears on hover you can use [`TooltipIcon`](https://carbon-components-svelte.onrender.com/components/TooltipIcon) or [`TooltipDefinition`](https://carbon-components-svelte.onrender.com/components/TooltipDefinition).

**Comment by metonym** [comment]
Fixed in [v0.101.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.101.1)
